admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,216 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+_back up_ (saving state to separate storage)
+and _restore_ (recreating state from separate storage)
+at the cluster level.
+There is also some preliminary support for
+xref:project-backup[per-project backup].
+The full state of a cluster installation includes:
+
+- etcd data on each master
+- API objects
+- registry storage
+- volume storage
+
+This topic does not cover how to back up and restore
+link:../install_config/persistent_storage/index.html[persistent storage],
+as those topics are left to the underlying storage provider.
+
+
+[[backup-restore-prerequisites]]
+== Prerequisites
+
+Because the restore procedure involves a
+link:#cluster-restore[complete reinstallation],
+we recommend that you save all the files used in the initial installation.
```

Comment:
@tpoitras @tnguyen-rh "We" should defs not be used (first-person) and neither "it is" (expletive). I don't see what's wrong with saying the reader should just do it. So:

"Because the restore procedure involves a complete reinstallation, save all the files used in the initial installation. This maybe include:"

This may be used in a bullet point with the below "install various utility commands" command, I guess.